src/evaluation/liveRec/Config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<requirements>
+    <TwoEarsPart sub="src" startup="startBinauralSimulator">binaural-simulator</TwoEarsPart>
+    <TwoEarsPart branch="master"             startup="startAuditoryFrontEnd">auditory-front-end</TwoEarsPart>
```

Comment:
No, then it won't know which repositories are required. We just want to have flexibility (i.e. own knowledge) about which branches are used.